cypress/integration/visitorCanVisitTheApplication.feature.js

```diff
@@ -20,25 +20,18 @@ describe('Visitor can visit the application and', () => {
     it('is expected to see a collection of articles', () => {
       cy.get('[data-cy=article-collection]')
         .children()
-        .should('have.length', 20);
+        .should('have.length', 16);
     });
 
     it('is expected to see article title', () => {
       cy.get('[data-cy=article-collection]').children().first().within(() => {
-        cy.get('[data-cy=article-title]').should('contain', 'Latin News');
+        cy.get('[data-cy=article-title]').should('contain', 'The Doors of Perception');
       });
     });
 
-    it('is expected to see article body', () => {
+      it('is expected to see article creation date', () => {
       cy.get('[data-cy=article-collection]').children().first().within(() => {
-        cy.get('[data-cy=article-body]').should('contain', 'Lorem ipsum');
```

Comment:
You removed the spec to check the body, but you should have change it to check the teaser now